db.js

```diff
@@ -0,0 +1,51 @@
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+module.exports = {
+    getDishes,
+    addDish,
+    getDish,
+    getRecipes,
+    addRecipe,
+    getRecipe,
+    getShoppingList
+};
+
+function getDishes(){
+    return db('dishes')
+}
+
+function getDish(id){
+    return db('dishes').where({ id: Number(id) });
```

Comment:
Adding .first() here will take the dish out of an array so you return only the object with the dish you want